app/views/datasets/_meta_data.html.erb

```diff
@@ -36,9 +36,11 @@
             <span itemprop="name"><%= dataset_location(@dataset) %></span>
           </dd>
 
-          <% if @dataset.topic %>
-            <dt><%= t('.topic') %>:</dt>
-            <dd><%= @dataset.topic['title'] %>
+          <dt><%= t('.topic') %>:</dt>
+          <% if(@dataset.topic) %>
+            <dd><%= @dataset.topic['title'] %></dd>
```

Comment:
is there a method that wouldn't crash if the topic didn't have a title (from bad input data). Python has `dataset.topic.get('title', '')` that provides a default

Reply:
In other words we need not to show the user a 500 page, unless it's actually impossible to show anything useful.

Reply:
`<%= @dataset.topic['title'] %>` will return the default value, in this case `nil` (and won't crash) if the `topic` hash has no `title` key. See https://docs.ruby-lang.org/en/2.0.0/Hash.html#method-i-5B-5D

The equivalent to `get` in Python would be `fetch`. See https://docs.ruby-lang.org/en/2.0.0/Hash.html#method-i-fetch